### PR TITLE
Feeds list page with filter and sorting

### PR DESCRIPTION
**Pull request by Xpirix**
- #20 

Please find below a screenshot for the Feed list page:
Screenshot 2023-10-16 at 11 52 42 AM

### CODE REVIEW

File: qgisfeedproject/qgisfeed/views.py [hunk 1]
```diff
@@ -107,8 +110,72 @@ def get(self, request):
...
+
+
+    # Get sorting parameters from the query string
+    sort_by = request.GET.get('sort_by', 'publish_from')
+    order = request.GET.get('order', 'desc')
+    current_order = order
+
+    if order == 'asc':
+        feeds_entry = feeds_entry.order_by(sort_by)
+        order = 'desc'
```
Comment (dimasciput) [inline qgisfeedproject/qgisfeed/views.py:149]: It's bit confusing here, probably just use variable 'next_order'

**Review by dimasciput — CHANGES_REQUESTED** [review]
@Xpirix  LGTM. Could you please make one small change.

File: qgisfeedproject/qgisfeed/views.py (hunk 1 above)
Comment (Xpirix) [reply qgisfeedproject/qgisfeed/views.py]: @dimasciput Tested ACK. You're right, I've fixed it.

File: qgisfeedproject/templates/layouts/base-fullscreen.html [hunk 2]
```diff
@@ -50,12 +50,15 @@
       <div id="menu" class="navbar-menu">
           <div class="navbar-end">
               
-                  
-                  
-                  <a class="navbar-item has-text-weight-bold is-size-7 is-active" href="/accounts/login">
-                      <span>Login</span>
-                  </a>
-                  
+            {% if user.is_authenticated and user.is_staff %}
```
Comment (dimasciput) [inline qgisfeedproject/templates/layouts/base-fullscreen.html:53]: @Xpirix, why can only staff logout?

Comment (Xpirix) [reply qgisfeedproject/templates/layouts/base-fullscreen.html:53]: @dimasciput , As in the administration page, only superuser (`is_superuser=True`, `is_staff=True`) or staff(`is_superuser=False`, `is_staff=True`) users can log in to the management page. However, there is the default user called 'qgis_user' that is authenticated automatically but is not an admin or staff. Thus, only authenticated admin and staff users can log in and see the logout button.

image

Comment (Xpirix) [reply qgisfeedproject/templates/layouts/base-fullscreen.html:53]: @dimasciput , I think this needs to be fixed in the future beacause if we remove the condition `user.is_staff`, we will always see the logout button even if there is no authenticated user.

Comment (dimasciput) [reply qgisfeedproject/templates/layouts/base-fullscreen.html:53]: Ah ok, yes we can re-visit this later.

**Pull request merged**